### DISCUSSION TITLE
How to work with colors?

### DISCUSSION THREAD

**Discussion by radoslavkarlik** [saadeghi/daisyui#2528] [Q&A] [answered]
I am trying to learn how to design pages for my personal projects and I struggle when it comes to colors. It's usually recommended to have 3 colors: primary, secondary, and accent. With like 60:30:10 ratio. DaisyUI suggests using semantic names for colors like primary. So let's say I have a primary color background and I want to have 2 buttons there: one is primary and one is secondary. The primary color button looks horrible on the primary color background. So how do you work with this then? I can't wrap my head around it and it makes the semantic colors unusable for me.

**Comment by saadeghi** [comment, marked as answer]
With 60-30-10 practice you can use base `base` colors (`base-100`, `base-200`, `base-300`) for 60% of screen, then you can use a dark neutral color (`nautral` or `base-content`) for 30% of screen (including text content) and finally use one or 2 of the brand colors for the remaining 10%.

The word "primary" has been used for different things in design articles. It doesn't necessarily mean the "primary" color in daisyUI palette. Basically the 3 colors primary, secondary, accent are "brand colors" in daisyUI and it's usually better of you keep them under 10% combined because overusing these 3 saturated colors might make your app look unprofessional.

A common practice is, first design everything in black and white (for example with `base-100` and `neutral` then depending on the context, replace some of the `neutral` parts with `primary` if it needs to be more eye catching.

Let's say you're designing a landing page, you can sue `base-*` for background, you can use `neutral` for navbar and `base-content` for the text. And then you put a single call-to-action button with primary color.

Generally don't try to use all the colors. Some UIs are better with only base and neutral, some with 3 colors, base, neutral, primary.

And at the end, 60-30-10 rule is not a rule, it's just a practice and it's not something all designers agree on.

Let me know if you have any questions.
